src/endpoints/comparisons/types.ts

```diff
@@ -34,6 +41,10 @@ export type ComparisonSide = {
     fileType: FileType;
     sourceURL?: string | null;
     displayName?: string | null;
+    pageCount?: number | null;
+    originalDocumentSizeBytes?: number | null;
+    highlightedDocumentSizeBytes?: number | null;
+    file?: string | null;
 };
 
 export type FileType = keyof typeof allowedFileTypes;
```

Comment:
another reference to `allowedFileTypes` that now doesn't match the casing of the other reference